ironfish/src/rpc/routes/chain/followChain.ts

```diff
@@ -125,6 +159,17 @@ router.register<typeof FollowChainStreamRequestSchema, FollowChainStreamResponse
             spends: transaction.spends.map((spend) => ({
               nullifier: spend.nullifier.toString('hex'),
             })),
+            mints: transaction.mints.map((mint) => ({
+              id: mint.asset.id().toString('hex'),
+              metadata: BufferUtils.toHuman(mint.asset.metadata()),
+              name: BufferUtils.toHuman(mint.asset.name()),
```

Comment:
hex these and do this on the other side, we just ran into issues with this today.

Reply:
Leaving as is for consistency with block graffiti